services/frontend/react_app/src/pages/home.tsx

```diff
@@ -148,23 +187,45 @@ export const HomePage = () => {
                                     >
                                         Request Ride
                                     </Button>
-                                    {/*<Text color='gray' mt={4}>The distance between the two points is 40 miles</Text>*/}
                                 </Box>
                             </Stack>
                         </CardBody>
                     </Card>
+
+                    {lastRequestedDrive.driverArrival &&
+                        <HStack divider={<StackDivider/>} spacing='4' justifyContent="center" alignItems="center"
+                                bg="teal.200" p="1rem" maxW={600} borderRadius={4}>
+                            <Countdown
+                                date={new Date(Date.now()).setSeconds(lastRequestedDrive.driverArrival)}
+                                renderer={(props) => countdownRenderer({
+                                    ...props,
+                                    driverName: lastRequestedDrive.driverName
+                                })}
+                            />
+                        </HStack>}
                 </Stack>
                 <Stack flexGrow={1} justifyContent='space-between' w='50%' h='100%' maxH={'900px'}>
-                    <Stack w='100%' backgroundColor='white' overflowY='auto'
-                           maxH={'50%'}>
-                        <Logs logs={logs}/>
-                    </Stack>
+                    <div className={`${styles.drawer} ${logsModal.isOpen ? styles.open : ''}`}>
+                        <div className={styles.drawerHeader}>
+                            <Heading size="md">Notifications Logs</Heading>
```

Comment:
Should just say "logs" and have a margin above. Also, can you make the border between each set of log entries darker?

Currently seems very subtle and hard to see:

<img width="625" alt="Screenshot 2024-12-23 at 8 32 37 AM" src="https://github.com/user-attachments/assets/0846da7d-69b0-4315-bae9-cc3f320a9fbd" />